docs/operations-guide/environment-variables.md

```diff
@@ -1,61 +1,810 @@
+
+Type: string (`"tls"`, `"ssl"`, `"starttls"`, `"none"`)<br>
+Default: `"none"`
+
+SMTP secure connection protocol.
+
+#### `MB_EMAIL_SMTP_USERNAME`
+
+Type: string<br>
+Default: `null`
+
+SMTP username.
+
+#### `MB_EMBEDDING_APP_ORIGIN`
+
+Only available in Enterprise Edition<br>
+Type: string<br>
+Default: `null`
+
+URL of origin allowed to embed the full Metabase application.
+
+#### `MB_EMOJI_IN_LOGS`
+
+Type: boolean<br>
+Default: `true`
+
+Emojis on log lines. When set to `false` it will disable log line emojis. This is disabled on Windows. Related to [MB_COLORIZE_LOGS](#MB_COLORIZE_LOGS).
+
+#### `MB_ENABLE_EMBEDDING`
+
+Type: boolean<br>
+Default: `false`
+
+Allow admins to securely embed questions and dashboards within other applications?
+
+#### `MB_ENABLE_NESTED_QUERIES`
+
+Type: boolean<br>
+Default: `true`
+
+Allow using a saved question as the source for other queries?
+
+#### `MB_ENABLE_PASSWORD_LOGIN`
+
+Only available in Enterprise Edition<br>
+Type: boolean<br>
+Default: `true`
+
+Allow logging in by email and password, when SSO login options are enabled.
+
+#### `MB_ENABLE_PUBLIC_SHARING`
+
+Type: boolean<br>
+Default: `false`
+
+Enable admins to create publicly viewable links (and embedded iframes) for Questions and Dashboards?
+
+#### `MB_ENABLE_QUERY_CACHING`
+
+Type: boolean<br>
+Default: `false`
+
+Enabling caching will save the results of queries that take a long time to run.
+
+#### `MB_ENABLE_TEST_ENDPOINTS`
+
+Type: boolean<br>
+Default: `null`
+
+When `true`, this will enable `/api/testing` endpoint. **Warning:** This should never be enabled in production system.
+
+#### `MB_ENABLE_XRAYS`
+
+Type: boolean<br>
+Default: `true`
+
+Allow users to explore data using X-rays
+
+#### `MB_ENCRYPTION_SECRET_KEY`
+
+Type: string<br>
+Default: `null`
+
+When set, this will encrypt database credentials stored in the application database. Requirement: minimum 16 characters base64-encoded string.
+
+Also see documentation page [Encrypting database details at rest](encrypting-database-details-at-rest.md).
+
+#### `MB_GOOGLE_AUTH_AUTO_CREATE_ACCOUNTS_DOMAIN`
+
+Type: string<br>
+Default: `null`
+
+When set, allow users to sign up on their own if their Google account email address is from this domain.
+
+#### `MB_GOOGLE_AUTH_CLIENT_ID`
+
+Type: string<br>
+Default: `null`
+
+Client ID for Google Auth SSO. If this is set, Google Auth is considered to be enabled.
+
+#### `MB_JDBC_DATA_WAREHOUSE_MAX_CONNECTION_POOL_SIZE`
+
+Type: integer<br>
+Default: `15`<br>
+Since: 0.35.0
+
+Maximum number of connections to the data source databases. The maximum is for each database setup in Admin Panel > Databases, not a total for all databases.
+
+Change this to a higher value, if you notice that regular usage consumes close to or all connections. When all connections are in use, then Metabase will be slower to return results for queries, since it would have to wait for an available connection before processing the next query in the queue.
```

Comment:
```suggestion
Change this to a higher value if you notice that regular usage consumes all or close to all connections. When all connections are in use then Metabase will be slower to return results for queries, since it would have to wait for an available connection before processing the next query in the queue.
```